webroot/adm/oncall/generate-api-key-secret.html

```diff
@@ -0,0 +1,661 @@
+                                <input type="text" class="form-control" id="participantName"
+                                    placeholder="Leave empty for all participants / clients">
+                            </div>
+                        </div>
+                        <div class="col">
+                            <a href="#" class="btn btn-primary" id="searchParticipant">Search</a>
+                        </div>
+                    </div>
+                    <div class="row pt-3">
+                        <div class="col">
+                            <h5>Search Results:</h5>
+                        </div>
+                    </div>
+                    <div class="row pt-3 px-2 collapse" id="searchSiteNoSiteFoundAlert">
+                        <div class="alert alert-warning d-flex align-items-center pt-3" role="alert">
+                            <svg class="bi flex-shrink-0 me-2" width="24" height="24" role="img" aria-label="Warning:">
+                                <use xlink:href="#exclamation-triangle-fill" />
+                            </svg>
+                            <div>
+                                No site found
+                            </div>
+                        </div>
+                    </div>
+                    <div class="row pt-3 px-2 collapse" id="searchSiteErrorAlert">
+                        <div class="alert alert-danger d-flex align-items-center pt-3" role="alert">
+                            <svg class="bi flex-shrink-0 me-2" width="24" height="24" role="img" aria-label="Warning:">
+                                <use xlink:href="#exclamation-triangle-fill" />
+                            </svg>
+                            <div id="searchOutputErrorDiv">
+                            </div>
+                        </div>
+                    </div>
+
+                    <div class="row px-2 table-responsive">
+                        <table class="table table-bordered" id="searchResultsTable">
+                            <thead>
+                                <th scope="col" class="col-1">Sites</th>
+                                <th scope="col" class="col-1">Clients</th>
+                            </thead>
+                            <tbody id="searchResultsTableBody">
+                            </tbody>
+                        </table>
+                    </div>
+                </div>
+            </div>
+            <div class="row">
+                <div class="card card-body">
+                    <div class="row pt-1">
+                        <div class="col">
+                            <div class="row">
+                                <h5>Update Operations:</h5>
+                            </div>
+                            <div class="row py-2">
+                                <div>
+                                    Use this to create new site, update existing sites or create new clients.
```

Comment:
Similar to above, I don't think "clients" makes sense here. Either "client keys" or "API keys"